src/components/Questionnaire/structured/handlers.ts

```diff
@@ -149,7 +149,7 @@ const handlers: {
   },
   appointment: {
     getRequests: (appointment, { facilityId, patientId }) => {
-      const { reason_for_visit, slot_id } = appointment[0];
+      const { reason_for_visit = "", slot_id = "" } = appointment[0] ?? {};
```

Comment:
this is not good... this is just making things more ugly...

slot_id should never be "" in the first place.